packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -134,7 +139,6 @@ function unwindInterruptedWork(interruptedWork: Fiber) {
       break;
     case DehydratedSuspenseComponent:
       if (enableSuspenseServerRenderer) {
-        // TODO: popHydrationState
```

Comment:
This one is not needed since restarting the root covers it. I think anything covered by resetFreshStack also doesn't need to be here.